libs/knowledge-store/ragstack_knowledge_store/cassandra.py

```diff
@@ -512,7 +512,7 @@ def similarity_search_by_vector(
     def _query_by_ids(
         self,
         ids: Iterable[str],
-    ) -> Iterable[Document]:
+    ) -> List[Document]:
```

Comment:
Should these stick with `Iterable`? Wouldn't that let them change implementation more easily?

Reply:
It's an internal method so the signature can be changed later if needed.

Reply:
Yes, it 's better to keep `Iterable` to evolve it more easily.
Done.